pandas/tests/arrays/string_/test_string.py

```diff
@@ -523,7 +523,7 @@ def test_memory_usage(dtype):
     # GH 33963
 
     if dtype.storage == "pyarrow":
-        pytest.skip("not applicable")
+        pytest.skip(f"not applicable for {dtype.storage}")
```

Comment:
this is a never-expected-to-change thing, i.e. shouldnt be an xfail?

Reply:
I assume so since I think that would require a pyarrow API to get memory usage (didn't see one quickly skimming)